### DISCUSSION TITLE
New images?

### DISCUSSION THREAD

**Discussion by ghost** [matt8707/hass-config#160] [Q&A]
Hello @matt8707,

I really like your design. I just started to work with it and found that a lot of devices I would use don't have images by default in the package, like motion sensor, smart wall socket. 
Do you plan to add new images any time soon?

Many thanks and congrats again for your work.

/Sandor

**Comment by texasmouse** [comment]
Personally, I tried and failed at making new icons and I can't figure out why.  That led me to temporarily give up and look for others that were successful.  I looked through pretty much all of the code forks to see what others did.  I found the following forks have great icon files.  

- lulu195
- tnusch
- trentbraidner

If anyone finds other cut-and-paste sources, please share them here.

**Comment by CokACola44** [comment]
Hello everyone,

Thank you for sharing this awesome project @matt8707 !
I'm also limited by the default icons in the package. I tried to add animated icons in the YAML file but I don't know anything about SVG, I didn't succeed. If someone has icons, or a solution, I'm interested! Thanks :)
(And thanks to @texasmouse for sharing his research!)

**Comment by tnusch** [comment]
Hey @f0ld1s, @texasmouse, @CokACola44 ,

what you essentially need is the path of the SVG icon. 
a) Search for a material design icon here: https://pictogrammers.com/library/mdi/
b) See if you can find a publicly available SVG path for you icon, e.g. inside the .vue files here: https://unpkg.com/browse/mdi-vue@1.2.3/ 
c) Copy the SVG path into your icons.yaml, similarly how it's done in the main repository or in our forks.

Hope this helps! 

Best regards,
Tim
